authors/settings.py

```diff
@@ -80,9 +80,13 @@
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("DB_NAME"),
```

Comment:
You should include defaults for all these variables. 
```suggestion
        "NAME": os.environ.get("DB_NAME", "authors_haven_db"),
```

Reply:
@archibishop  please check the changes.